src/resampling.jl

```diff
@@ -363,7 +389,9 @@ julia> f = resample_nfft(a, new_pos, is_src_coords=false);
 julia> @ve a e f
 ```
 """
-function resample_nfft(img::AbstractArray{T,D}, new_pos::AbstractArray{T2,D2}, dst_size=nothing; pad_value=nothing, is_in_pixels=false, is_local_shift=false, is_src_coords=true, reltol=1e-9)::AbstractArray{T,D} where {T,D,T2,D2} # 
+function resample_nfft(img::AbstractArray{T,D}, new_pos::AbstractArray{T2,D2}, dst_size=nothing; 
+                        pad_value=nothing, is_in_pixels=false, is_local_shift=false, is_src_coords=true, 
+                        reltol=1e-9)::AbstractArray{T,D} where {T,D,T2,D2} # 
```

Comment:
is it ok to use a `1e-9` which is Float64?
Wouldn't it be safer to use `T(1e-9)`?

Reply:
no. It should be preserving the type and T can anyway be complex.